tests/integration/security/testdata/authz/v1beta1-log.yaml.tmpl

```diff
@@ -0,0 +1,48 @@
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-b-log
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "b"
+  action: AUDIT
+  rules:
+  - to:
+    - operation:
+        paths: ["/allow_log"]
+
+---
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-c-log
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "c"
+  action: AUDIT
+  rules:
+  - to:
+    - operation:
+        paths: ["/deny_log"]
+
+---
+
+apiVersion: "security.istio.io/v1beta1"
+kind: AuthorizationPolicy
+metadata:
+  name: policy-c-deny
+  namespace: "{{ .Namespace }}"
+spec:
+  selector:
+    matchLabels:
+      "app": "c"
+  action: DENY
```

Comment:
Can you also add a new case that both LOG and ALLOW policy are specified for the same workload? Similar to the case DENY policy here.